DeepNeuralNetworks/DNN.ipynb

```diff
@@ -1,1399 +1,2548 @@
 {
```

Comment:
<p>For a lot of these examples the training data is REALLY dense, so it's guaranteed to learn the space you're showing off, maybe for illustrative purposes it would make more sense to have more sparse data? (Or at least increase the transparency on this graph and show the training data with different markers). Just adding some random noise would make this seem a little bit more useful </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>